pulp_2_tests/constants.py

```diff
@@ -520,6 +520,9 @@
 RPM_INVALID_URL = urljoin(RPM_INVALID_FEED_URL, 'invalid.rpm')
 """The URL to an invalid RPM package."""
 
+RPM_KICKSTART_FEED_URL = urljoin(PULP_FIXTURES_BASE_URL, 'rpm-kickstart/')
+"""The URL to a KICKSTART repository."""
+
```

Comment:
What about add a note stating that this fixtures are not generated using pulp-fixtures? Something like...

```.rst
.. NOTE:: This repository is not generated by `pulp-fixtures`_.
```